### PR TITLE
Added protobuf V1 control measure log data parsing

### PR DESCRIPTION
**Pull request by jessemapel**

### CODE REVIEW

**Comment by jessemapel** [comment]
[Parse the version 1 log data](https://trello.com/c/z4mXPAfr/137-parse-the-version-1-log-data)

**Comment by jessemapel** [comment]
Don't merge me in yet. I need to merge in Adam's stuff first

File: isis/src/control/objs/ControlNetVersioner/ControlNetVersioner.cpp [hunk 1]
```diff
@@ -849,7 +849,7 @@ namespace Isis {
 
     // Create the header
     try {
-      ControlNetHeaderV0001 header;
+      ControlNetHeaderV0002 header;
```
Comment (kberryUSGS) [inline isis/src/control/objs/ControlNetVersioner/ControlNetVersioner.cpp]: Why did you change this to `V0002` ?

Comment (jessemapel) [reply isis/src/control/objs/ControlNetVersioner/ControlNetVersioner.cpp]: Because technically the protobufv1 stuff is for control net v2.

**Comment by jessemapel** [comment]
Ready to be merged now

File: isis/src/control/objs/ControlNetVersioner/ControlNetVersioner.cpp (hunk 1 above)
Comment (kberryUSGS) [reply isis/src/control/objs/ControlNetVersioner/ControlNetVersioner.cpp]: Ah, right thanks.

**Comment by kberryUSGS** [comment]
:+1:
